dev/com.ibm.ws.security.jwt/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -106,4 +106,19 @@ elapsedNBF.desc=Identifies the elapsed time until JWT will start to be accepted
 
 #Do not translate "amr"
 amrValues=Authentication methods references in string array
-amrValues.desc= Specifies Authentication Methods References so it can enforce step-up authentication and know how the user is authenticated. Specify a comma separated list of valid amr claim values to be used.
\ No newline at end of file
+amrValues.desc= Specifies Authentication Methods References so it can enforce step-up authentication and know how the user is authenticated. Specify a comma separated list of valid amr claim values to be used.
+
+# Do not translate "Content Encryption Key"
+keyManagementKeyAlgorithm=Key management key algorithm
+keyManagementKeyAlgorithm.desc=Specifies the encryption algorithm that is used to encrypt the Content Encryption Key of a JWE.
```

Comment:
**change**
...the Content Encryption Key of a JWE.
**to**
...the content encryption Key of a JWE.

Reply:
"Content Encryption Key" is a term defined in the JWE specification that we're implementing here (https://tools.ietf.org/html/rfc7516#section-2); the whole phrase is a specific term that refers to a well-defined object.